src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -919,17 +919,33 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
         }
     }
 
+    /**
+     * Asserts that the server returns an 'item-not-found' error response when
+     * deleting a node that does not exist.
+     *
+     * <p>
```

Comment:
I think this reference is incorrect? https://xmpp.org/extensions/xep-0060.html#owner-delete-error-node